scripts/asm-installer/install_asm

```diff
@@ -1411,6 +1444,23 @@ is_workload_identity_enabled() {
   if [[ "${ENABLED}" = 'null' ]]; then false; fi
 }
 
+get_hub_workload_identity() {
+  local ID_NUMBER
+  ID_NUMBER=`kubectl get membership -o=json | jq '.items | length'`
+  if [[ ${ID_NUMBER} -gt 1 ]]; then
+    fatal "Found more than one Hub Membership. Please make sure one cluster is associated with one Hub Membership.";
+  fi
+  local IDENTITY_PROVIDER
+  IDENTITY_PROVIDER=`kubectl get membership -o=json | jq .items[0].spec.identity_provider`
```

Comment:
It looks like you're calling `kubectl get membership` twice. Can we do something like this?

MEMBERSHIPS="$(kubectl get membership -o=json | jq '.items')"
NUM_MEMBERSHIPS="$(echo "${MEMBERSHIPS}" | jq 'length')"
if $NUM_MEMBERSHIPS -gt 1...
...
IDENDITY_PROVIDER = "$(echo "${MEMBERSHIPS}" | jq .[0].spec.identity_provider)"